deck-core/src/main/kotlin/com/deck/core/cache/CacheManager.kt

```diff
@@ -0,0 +1,28 @@
+package com.deck.core.cache
+
+import com.deck.common.util.GenericId
+import com.deck.core.entity.Channel
+import com.deck.core.entity.User
+import com.github.benmanes.caffeine.cache.Cache
+import java.util.*
+
+public interface CacheManager {
```

Comment:
Isn't `putEntity` and `updateEntity` the same thing?

Reply:
Do you think i'd better remove the update and leave the put?